### ISSUE TITLE
 initElapsedS &  readElapsedS calculation right?

### ISSUE THREAD

**Issue by WhaoSH** [UoB-HPC/BabelStream#196]
In main.cpp #342 #343, the calculation of   initElapsedS &  readElapsedS is confusing, why initElapsedS=(read2 - read1), while readElapsedS=(init2 - init1)?
```c++
 auto initElapsedS = std::chrono::duration_cast<std::chrono::duration<double>>(read2 - read1).count();
 auto readElapsedS = std::chrono::duration_cast<std::chrono::duration<double>>(init2 - init1).count();
```

**Comment by tomdeakin** [comment]
Thanks for reporting this - we have a bug fix coming in a PR.

**Comment by tomdeakin** [comment]
Will be fixed by #186

**Comment by gonzalobg** [comment]
Fixed by #186 .
